src/main/resources/github.properties.example

```diff
@@ -1 +1 @@
-github.token=baadf00d
\ No newline at end of file
+github.token=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
```

Comment:
Święta zasada programistów: nigdy nie commitujemy do repozytorium rzeczy wrażliwych typu tokeny lub hasła :) Stąd właśnie w rozwiązaniu użyty jest plik github.properties, który celowo jest dołożony do .gitignore - i to tam trzeba było umieścić ten token